packages/components/src/core/ContentCard/index.tsx

```diff
@@ -205,12 +205,12 @@ const ContentCard = React.forwardRef<HTMLDivElement, ContentCardProps>(
       >
         {clickableCard ? (
           <StyledCardActionArea
+            {...clickableCardProps}
```

Comment:
can we add a comment why this spread needs to happen at the top instead of being at the bottom, so the user can overwrite the default props please 🙏 ? Thank you!

Reply:
Absolutely 🙃🙂